targets/targets.json

```diff
@@ -47,6 +47,11 @@
             "tickless-from-us-ticker": {
                 "help": "Run tickless from the microsecond ticker rather than the low power ticker. Running tickless off of the microsecond ticker improves interrupt latency on targets which use lpticker_delay_ticks",
                 "value": false
+            },
+            "hw-watchdog_timeout": {
+                "help": "Define the timeout in ms value LowPowerTicker to do HW kick",
```

Comment:
is this low power ticker? I can see using wait with this macro and could not find reference using LowPowerTicker in this PR?

why does it have `macro_name` ? if it follows the default name , would be with prefix `MBED_CONF_TAR` or simarl to distinguish this comes from target